enclave-tls/src/Makefile

```diff
@@ -1,18 +1,21 @@
-TOPDIR ?= ..
+TOPDIR ?= $(shell readlink -f ..)
```

Comment:
Split this change as the first commit, and move all wolfssl tls wrapper specific codes into 2nd commit.